samples/bluetooth/fast_pair/locator_tag/src/fp_adv.c

```diff
@@ -59,8 +65,26 @@ static struct bt_le_adv_param fp_adv_param = {
 };
 
 static void fp_adv_restart_work_handle(struct k_work *w);
+static void fp_adv_rpa_suspension_work_handle(struct k_work *w);
 
 static K_WORK_DEFINE(fp_adv_restart_work, fp_adv_restart_work_handle);
+static K_WORK_DELAYABLE_DEFINE(fp_adv_rpa_suspension_work, fp_adv_rpa_suspension_work_handle);
+
+static void fp_adv_rpa_rotation_suspend(bool suspended)
+{
+	fp_adv_rpa_rotation_suspended = suspended;
+}
```

Comment:
If it is used only to assign the value consider removing this function. 
Or maybe consider adding a log, I think that it could be useful.

Reply:
I would keep it as part of the separate function definition to have a single place in the module when this flag value is changed. The debugging experience is also nice as you can add a single log to track the flag status in this function (and that's how I assessed the correctness of this solution).

@mkapala-nordic, I can add a debug-level log, which is not logged in the default configuration. The sample logs are already overwhelming and I would like to avoid extending them.

@alstrzebonski, agree regarding the name change